### PR TITLE
Fixing headings in the Overview section

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

* Checking Overview IDs uniqueness
* Adding missing _{context} attribute to assembly headings
* Adding context information for included files
* Adding :page-liquid: where needed
* Deleting unnecessary content (empty templates without its place in the doc plan https://github.com/eclipse/che/issues/14415)
* Reworking `Installing Che using storage classes` and placing it in *IG/ Advanced configuration options - Checked by Oleksandr Andriienko and Serhii Leshchenko *

### CODE REVIEW

File: src/main/pages/che-7/installation-guide/proc_installing-che-using-storage-classes.adoc [hunk 1]
```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//
+// advanced-configuration-options
+
+[id="proc_installing-{prod-id-short}-using-storage-classes_{context}"]
+ 
+= Installing {prod-short} using storage classes
+ 
+:context: proc_installing-{prod-id-short}-using-storage-classes
```
Comment (rkratky) [inline src/main/pages/che-7/installation-guide/proc_installing-che-using-storage-classes.adoc]: The context should not include "proc_".
```suggestion
:context: installing-{prod-id-short}-using-storage-classes
```

Comment (rkratky) [reply src/main/pages/che-7/installation-guide/proc_installing-che-using-storage-classes.adoc]: Just realized... this a module included from an assembly. It shouldn't set its own context at all.
```suggestion
```

File: src/main/pages/che-7/overview/assembly_accessing-che-from-openshift-developer-perspective.adoc [hunk 2]
```diff
@@ -8,15 +8,17 @@ folder: che-7/overview
 summary:
 ---
 
+:page-liquid:
+
```
Comment (rkratky) [inline src/main/pages/che-7/overview/assembly_accessing-che-from-openshift-developer-perspective.adoc:13]: Parent context should be set.
```suggestion
 :parent-context-of-accessing-che-from-openshift-developer-perspective: {context}
 
```

Comment (rkratky) [reply src/main/pages/che-7/overview/assembly_accessing-che-from-openshift-developer-perspective.adoc:13]: Also, reset the context at the end of the assembly with:
```
:context: {parent-context-of-accessing-che-from-openshift-developer-perspective}
```
(I can't add it as a suggestion.)

**Review by rkratky — CHANGES_REQUESTED** [review]
Nice job, @MichalMaler -- good to have consistency. A few comments added.

**Comment by MichalMaler** [comment]
Hi, @rkratky Thx for the quick and kind feedback! Suggestions are on the way

**Review by rkratky — APPROVED** [review]

**Pull request merged**